src/main/webapp/scripts/session-script.js

```diff
@@ -31,6 +56,24 @@ function addOnClickListenerToElements() {
   });
 }
 
+/**
+ * function changeElementsToReadOnly() changes the two inputs,
+ * one on the welcome message and the other in the session 
+ * information div, to show the session ID and then changes them
+ * to read only (meaning they cannot be changed once set).
+ * @param {string} sessionId
+ */
+function changeElementsToReadOnly(sessionId) {
```

Comment:
nit: this does more than change to readOnly, it also sets values, could you rename this to reflect that?

setReadOnlyMessages or something like that.

Alternatively, you could make two methods, a setValue() and a setReadOnly() method.